test/test-parser.js

```diff
@@ -1544,12 +1544,16 @@ describe('errors coverage', () => {
     assertThrows(callback, expectedMessage, { startsWith: true });
   });
 
+  //////////////////////////////////////////////////////////////////////////////
```

Comment:
Missed adding these error-code-block delimiters to the parser tests before. Sneaking them in.